$NetBSD: patch-ab,v 1.4 1999/12/16 00:58:27 dmcmahill Exp $

Set optimisation level appropriately

Don't do additional checksums of the source (gets thrown 
off by our patches, plus we already have files/md5)

--- ./src/makefile.orig	Wed Feb 25 08:18:04 1998
+++ ./src/makefile	Tue Dec  7 18:15:45 1999
@@ -3,5 +3,5 @@
 .SUFFIXES: .c .o
 CC = cc
-CFLAGS = -O
+CFLAGS = -O2
 SHELL = /bin/sh
 YACC = yacc
@@ -25,5 +25,5 @@
 OBJECTS = $(OBJECTSd) $(MALLOC)
 
-all: xsum.out f2c
+all: f2c
 
 f2c: $(OBJECTS)
